src/onboarding/ha-onboarding.ts

```diff
@@ -0,0 +1,203 @@
+import "@polymer/paper-input/paper-input";
+import "@material/mwc-button";
+import {
+  LitElement,
+  CSSResult,
+  css,
+  html,
+  PropertyValues,
+  property,
+  customElement,
+  TemplateResult,
+} from "lit-element";
+import { genClientId } from "home-assistant-js-websocket";
+import { litLocalizeLiteMixin } from "../mixins/lit-localize-lite-mixin";
+import { OnboardingStep, onboardUserStep } from "../data/onboarding";
+import { PolymerChangedEvent } from "../polymer-types";
+
+@customElement("ha-onboarding")
+class HaOnboarding extends litLocalizeLiteMixin(LitElement) {
+  public translationFragment = "page-onboarding";
+
+  @property() private _name = "";
```

Comment:
are these not initialized if you set the type to string?

Reply:
nope, it's undefined